docs/utilities.rst

```diff
@@ -1286,6 +1286,47 @@ hexidecimal digits.
     '0001..1e1f'
 
 
+Numeric Utils
+~~~~~~~~~~~~~
+
+``clamp(lower_bound, upper_bound, value)`` -> result
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Returns ``value`` clamped within the inclusive range defined by ``[lower_bound,
+upper_bound]``.  The value can be any number type that supports ``<`` and ``>``
+comparisons against the provided bounds.
+
+.. doctest::
+
+   >>> from eth_utils import clamp
+   >>> clamp(5, 8, 4)
+   5
+   >>> clamp(5, 8, 5)
+   5
+   >>> clamp(5, 8, 6)
+   6
+   >>> clamp(5, 8, 7)
+   7
+   >>> clamp(5, 8, 8)
+   8
+   >>> clamp(5, 8, 9)
+   8
```

Comment:
*nit* I think an upper limit of 7 is sufficient to demonstrate, and slims the example down by a line.